query/search/sort.go

```diff
@@ -0,0 +1,90 @@
+package search
```

Comment:
as sorting would be applicable for read queries as well, let's move sorting to its own package.